include/groonga/db.h

```diff
@@ -40,6 +40,41 @@ GRN_API grn_obj *
 grn_db_open(grn_ctx *ctx, const char *path);
 GRN_API void
 grn_db_touch(grn_ctx *ctx, grn_obj *db);
+/**
+ * \brief Checks the passed database and recovers it if it is broken and it
```

Comment:
[nits] 三単現のsがないほうが多いので合わせてあげられると良さそうです。
```suggestion
 * \brief Check the passed database and recover it if it is broken and it
```